newrelic-integration-e2e/internal/docker/docker.go

```diff
@@ -1,17 +1,44 @@
 package docker
 
 import (
+	"fmt"
 	"os"
 	"os/exec"
+	"strings"
 )
 
 const (
 	dockerComposeBin = "docker-compose"
 )
 
-func DockerComposeUp(path string) error{
-	cmd := exec.Command(dockerComposeBin, "-f", path, "up", "-d")
+
+func DockerComposeRun(path string,container string, envVars map[string]string) error {
+	DockerComposeBuild(path,container)
+	args := []string{"-f", path,"run"}
+	for k, v := range envVars {
+		args = append(args, "-e", fmt.Sprintf("%s=%s", k, v))
+	}
+	args = append(args,  "-d", container)
+	cmd := exec.Command(dockerComposeBin, args...)
+	cmd.Stdout = os.Stdout
```

Comment:
should we create a function to run commands?

Reply:
If you are open to help libraries, there a lot to manage running commands https://github.com/go-cmd/cmd

Reply:
executeOSCommands itself?